installation.md

```diff
@@ -55,6 +55,7 @@ This guide provides steps to run the MonComptePro Node.js application locally wh
 After setting up the application, start the Node.js server with:
 
 ```bash
+cd src
```

Comment:
:wave: I admit I don't get why this is necessary, can you explain please?

Reply:
I was convinced that when I wasn't in src I had errors in my console. I think it was other errors because I've just redone a test outside src and my stack is running correctly. Thx !